linkatos/printer.py

```diff
@@ -14,3 +14,22 @@ def ask_confirmation(message, slack_client):
     bot_says(message['channel'],
              compose_explanation(message['url']),
              slack_client)
+
+
+def compose_url_list(url_cache_list):
+    if len(url_cache_list) == 0:
+        return "The list is empty"
+
+    list_message = "The list of urls to be confirmed is: \n"
+
+    for index in range(0, len(url_cache_list)):
```

Comment:
The right way to iterate over a collection returning index and value is as follows:

```python
for index, value in enumerate(url_cache_list):
    extra = "{} - {} \n".format(index, value['url'])
```

Or, my preferred way:

```python
intro = "The list of urls to be confirmed is: \n"
options = ["{} - {}".format(i, v['url']) for i, v in enumerate(url_cache_list)]

return intro + "\n".join(options)
```